runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -468,6 +470,27 @@ private void swapMenuEntry(int index, MenuEntry menuEntry)
 			{
 				swap("start-minigame", option, target, index);
 			}
+
+			if (target.equals("trader crewmember"))
+			{
+				switch (config.swapTradeCrewmember())
+				{
+					case TRADE:
+						swap("trade", option, target, index);
+						break;
+					case CHARTER:
+						swap("charter", option, target, index);
+						break;
+					case PREVIOUS_LOCATION:
+						swap("charter-to catherby", option, target, index);
+						swap("charter-to brimhaven", option, target, index);
+						swap("charter-to port khazard", option, target, index);
+						swap("charter-to corsair cove", option, target, index);
+						swap("charter-to musa point", option, target, index);
+						swap("charter-to port sarim", option, target, index);
+						break;
```

Comment:
also add `case OFF:` to this imo

Reply:
Wouldn't this be more correct then? Seeing as neither will do anything.

``` 
case OFF:
case default:
    break;
```

Reply:
Sure, but i don't think it will ever hit `default`